src/scalarstats.jl

```diff
@@ -240,6 +309,14 @@ function _zscore_chksize(X::AbstractArray, μ::AbstractArray, σ::AbstractArray)
     end
 end
 
+
+"""
+    zscore!([Z,] X, μ, σ)
+
+Compute the z-scores of an array `X` with mean `μ` and standard deviation `σ`. If
```

Comment:
Not sure what you mean. There's no function called `zscores`, there's only `zscore` (singular) and its yelling version (`zscore!`), unless I'm *extremely* oblivious.